.github/pull_request_template.md

```diff
@@ -1,16 +1,9 @@
 # Description
 
-Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.
+[description]
 
-Fixes # (issue)
+Fixes #(issue)
 
 ## Changes and Descriptions
 
 [List your changes here]
-
-# Checklist:
-
-- [ ] I have added Rust doc comments to structs, enums, traits and functions
-- [ ] I have made corresponding changes to the documentation
-- [ ] I have performed a self-review of my code
-- [ ] I have added tests that prove my fix is effective or that my feature works
```

Comment:
I am not sure whether I can agree with this removal. Personally, I go through this checklist for every PR but I suppose the reality is, that it is rather ignored and many PRs do not include any ticks.

Reply:
Same here, I like having this checklist for chores that might be forgotten.

Reply:
Yes, In some PRs, I also found myself reading it to check mentally if things were done.

But, I found 2 points against having this there (maybe some other in place?).
- Most of the time, some points do not apply to the PR meaning.
- Most of the time, PRs are merged without clicking done in all those points.

From my point of view, I think each PR has its own checklist and should be thought of specifically for the PR purpose. So I agree with adding a checklist, but I think should be done explicitly at that moment of creation for the content of that PR.